pkg/pod/status.go

```diff
@@ -281,7 +281,8 @@ func getFailureMessage(pod *corev1.Pod) string {
 	for _, status := range pod.Status.ContainerStatuses {
 		term := status.State.Terminated
 		if term != nil && term.ExitCode != 0 {
-			return fmt.Sprintf("%q exited with code %d (image: %q); for logs run: kubectl -n %s logs %s -c %s",
+			return fmt.Sprintf(`%q exited with code %d (image: %q); for logs run:
+kubectl -n %s logs %s -c %s`,
```

Comment:
Thanks for this. IIUC the default line break limit in the YAML marshaller is 80 characters, so this will help a bit but not fully solve the issue.
The PR that adds the option to set the line max length on `go-yaml`, so I'm not sure it will be merged any time soon. In any case we are not in control of the YAML marshalling, so we cannot pick which version of the library to use.
I see a few options here:
1. according to this [github comment](https://github.com/go-yaml/yaml/issues/166#issue-143761794), appending a `\n` at the end of the line is a valid workaround
2. do what you proposed here - which would work be a partial fix

My only concern with both options that it might break the logic in the CLI, which currently displays the command properly (I'm guessing they might use the json marshaller):

```
$ tkn tr describe a-pipeline-to-reproduce-issue-2029-c2p4z-first-and-only-t-6bz84
Name:        a-pipeline-to-reproduce-issue-2029-c2p4z-first-and-only-t-6bz84
Namespace:   default

🌡️  Status

STARTED       DURATION     STATUS
2 weeks ago   44 seconds   Failed

Message

"step-fail" exited with code 1 (image: "docker.io/library/busybox@sha256:b26cd013274a657b86e706210ddd5cc1f82f50155791199d29b9e86e935ce135"); for logs run: kubectl -n default logs a-pipeline-to-reproduce-issue-2029-c2p4z-first-and-only-t-7kcz9 -c step-fail
```

Reply:
I tested this patch:
```
diff --git a/pkg/pod/status.go b/pkg/pod/status.go
index 93ba0cb3..b8443d68 100644
--- a/pkg/pod/status.go
+++ b/pkg/pod/status.go
@@ -281,7 +281,7 @@ func getFailureMessage(pod *corev1.Pod) string {
        for _, status := range pod.Status.ContainerStatuses {
                term := status.State.Terminated
                if term != nil && term.ExitCode != 0 {
-                       return fmt.Sprintf("%q exited with code %d (image: %q); for logs run: kubectl -n %s logs %s -c %s",
+                       return fmt.Sprintf("%q exited with code %d (image: %q); for logs run: kubectl -n %s logs %s -c %s\n",
                                status.Name, term.ExitCode, status.ImageID,
                                pod.Namespace, pod.Name, status.Name)
                }
```

and it works fine.:
```
        - lastTransitionTime: "2020-03-31T16:36:51Z"
          message: |
            "step-fail" exited with code 1 (image: "docker.io/library/busybox@sha256:b26cd013274a657b86e706210ddd5cc1f82f50155791199d29b9e86e935ce135"); for logs run: kubectl -n default logs a-pipeline-to-reproduce-issue-2029-kgwr4-first-and-only-t-lk2gn -c step-fail
          reason: Failed
          status: "False"
          type: Succeeded
```

Also it causes no issues to `tkn`.

Reply:
I will make this change then, this looks good. Thanks @afrittoli ! :)

Reply:
I have made this change, PTAL.